tensorstore/kvstore/s3/object_metadata.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The TensorStore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TENSORSTORE_KVSTORE_S3_OBJECT_METADATA_H_
+#define TENSORSTORE_KVSTORE_S3_OBJECT_METADATA_H_
+
+/// \file
+/// Key-value store where each key corresponds to a S3 object and the value is
+/// stored as the file content.
+
+#include <stdint.h>
```

Comment:
Prefer the C++ standard headers.

Reply:
Used `#include <cstdint>` but this ended up being removed along with `ObjectMetadata`.